docs/repositories/commits.md

```diff
@@ -75,11 +75,12 @@ This area displays the quality gate status and an overview of the code quality m
 
 The **New Issues** and **Fixed Issues** tabs display the list of issues that the commit created or fixed.
 
-{%
-    include-markdown "./issues.md"
-    start="<!--issue-detail-start-->"
-    end="<!--issue-detail-end-->"
-%}
+Click the title of an issue to see the following information:
+
+-   The committer and date of the commit that introduced the issue
+-   The [tool that reported the issue](../getting-started/supported-languages-and-tools.md) and the estimated time to fix it
+-   What's the issue and how to solve it
+-   The programming language and category of the issue
```

Comment:
I think the programming languages we show are related to the tool and not the issues found. I’m not sure why we do that though